src/ios/view.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * test view
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { UIScreen } from 'UIKit/UIScreen';
+import { UIColor } from 'UIKit/UIColor';
+import { UIView } from 'UIKit/UIView';
+
+export default class View {
```

Comment:
You're default exporting `View`, but `UIScreen`, `UIColor`, and `UIView` are not default exported from their respective modules. If a module only has a single export (like this view module does), then it's only export should be a default export.